app/src/App.vue

```diff
@@ -306,6 +298,17 @@ export default class App extends Mixins(
     })
   }
 
+  mounted (): void {
+    // if there is stored NR data to process then affiliate it now
+    const nr = JSON.parse(sessionStorage.getItem('NR_DATA'))
+    if (nr) {
+      // Use the new "magic link routes" in the BRD to perform the affiliations and draft creations.
+      Navigate(this.magicLink(nr))
+      // clear NR data for next time
+      sessionStorage.removeItem('NR_DATA')
+    }
+  }
```

Comment:
This code is for the following flow:
- user was not logged in
- user tried to incorporate a numbered business, or user loaded their approved NR and tried to "incorporate now"
- user was redirected to log in
- user was redirected back to Namerequest UI
- this code now runs to complete the new business creation

I _think_ the magic links don't support numbered businesses. @JazzarKarim ?

I _think_ you will need to keep the old code for creating numbered businesses.

Reply:
Actually, I might be wrong here. This code is for a NR only. For a new numbered business, it looks like LEGAL_TYPE and REQUEST_ACTION_CD is saved to session storage instead, and a different flow is used.

So, this is probably OK.